tdrs-backend/tdpservice/parsers/schema_defs/tanf/t3.py

```diff
@@ -73,41 +170,81 @@
         Field(item="6", name='CASE_NUMBER', type='string', startIndex=8, endIndex=19,
               required=True, validators=[]),
```

Comment:
`RPT_MONTH_YEAR` and `CASE_NUMBER` have validators in the other record types. if they're included in `child_one` then they may not need to be included for `child_two` but it's probably easier to have them match